src/main/java/lesson1/MyException.java

```diff
@@ -0,0 +1,8 @@
+package lesson1;
+
+public class MyException extends Exception {
```

Comment:
We could extend from `RuntimeException` that allows us to nut add `throws MyException` to methods.

Reply:
Added new changes